components/livekit/device-select.tsx

```diff
@@ -16,7 +16,7 @@ type DeviceSelectProps = React.ComponentProps<typeof SelectTrigger> & {
   kind: MediaDeviceKind;
   track?: LocalAudioTrack | LocalVideoTrack | undefined;
   requestPermissions?: boolean;
-  onError?: (error: Error) => void;
+  onMediaDeviceError?: (error: Error) => void;
   initialSelection?: string;
   onActiveDeviceChange?: (deviceId: string) => void;
```

Comment:
Note that alongside this change, I renamed this prop from `onError` => `onMediaDeviceError` because `onError` was already defined on `SelectTrigger` and meant something else (the default `onerror` on `HTMLElement`). After the rename the resulting generated types are more straightforward.